mmdet/core/post_processing/bbox_nms.py

```diff
@@ -54,3 +56,77 @@ def multiclass_nms(multi_bboxes,
         keep = keep[:max_num]
 
     return dets, labels[keep]
+
+
+def fast_nms(multi_bboxes,
+             multi_scores,
+             multi_coeffs,
+             score_thr,
+             iou_thr,
+             top_k,
+             max_num=-1):
+    """Fast NMS in `YOLACT <https://arxiv.org/abs/1904.02689>`_. Fast NMS
```

Comment:
"""Fast NMS in \`YOLACT <https://arxiv.org/abs/1904.02689>\`_.

Fast NMS ...
"""